### ISSUE TITLE
SEO: optimize the site for people looking for a CMS

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#123]
Follow-up from https://github.com/backdrop-ops/backdropcms.org/issues/122

**Comment by docwilmot** [comment]
This is important. Backdrop.org needs:
- **user help pages**
- what you can do with Backdrop. (types of sites, types of customer/organization)
- features (that are important to users, not just developers)
- **a discussion forum** or link to the Reddit. 
- user focused language on the front, not the why use backdrop stuff
- the Dragon, and a spot of color on the front
- screenshots

I could start writing stuff, but I'd need to see our hear some movement to get it actually done first.

**Comment by sutibun** [comment]
\*raises hand\* ooh ooh, assign this to the monkey. I wanna fully lead this if no one minds.

Why?

> "SEO: optimize the site for people looking for a CMS"

is the same as saying

> "Make people come to a concert performed by a nobody"

It's easy to say but not easy to do unless you have some idea what to do. Plus, it's a lot of work if you're doing it properly.

Just so happens, the monkey has optimized sites before so this is not first time. (read: experienced)

Assuming you guys put the monkey in charge, here is the next step:

The monkey will do an in-depth research into what *exactly* searchers are looking for on search engines. No SEO campaign can begin without this. This will take awhile but monkey willing to do it.

A quick research
------

For fun, the monkey did a lil' secret digging and found that the following are sample phrases that Backdrop is attracting. (i.e. people typed these phrases to stumble unto the Bd site)

![image](https://cloud.githubusercontent.com/assets/14132746/11707003/3c287a2c-9ec4-11e5-9df1-14dd80148a5e.png)

Also, these are other phrases people are using to discover Bd site

![image](https://cloud.githubusercontent.com/assets/14132746/11707052/b3107658-9ec4-11e5-97b8-a39f6aae616d.png)

Not much value here but gives an indication what people are searching to get to the site.

**Comment by sutibun** [comment]
since haven't heard anything on this ticket and wanna be sure monkey doesn't work on something that won't be used/implemented.... 

is monkey assigned this?

**Comment by jenlampton** [comment]
Yes Monkey, this is all yours! :)  Please let us know what we can do to help you!

**Comment by jenlampton** [comment]
@docwilmot to answer about things we need on backdropcms.org:

> user help pages
What does that mean? 

> what you can do with Backdrop. (types of sites, types of customer/organization)
Yes!

> features (that are important to users, not just developers)
Yes!

> a discussion forum or link to the Reddit.
We have one, somewhere...  where do you propose we move it to?

> user focused language on the front, not the why use backdrop stuff
Okay, can you propose something?

> the Dragon, and a spot of color on the front
Okay, again, suggestions?

> screenshots
Yes please :)  Maybe also a video?

**Comment by sutibun** [comment]
> Please let us know what we can do to help you!

@jenlampton *Only* one simple request at the moment: can you implement issue 149? please...

**Comment by serundeputy** [comment]
@jenlampton recently ported [metatag](https://github.com/backdrop-contrib/metatag)

A big push/win for SEO would be to install and configure metatag on backdropcms.org

**Comment by jenlampton** [comment]
committed metatag to the master branch.
